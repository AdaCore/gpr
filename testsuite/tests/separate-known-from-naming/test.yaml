description:
    separate name defined in naming package.
driver: build_and_run
project_file: main
main: main
