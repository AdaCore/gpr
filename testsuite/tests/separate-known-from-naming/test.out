PACK-Functions_Test.ada: info: invalid name for unit 'PACK-Functions_Test', should have only alpha numeric characters
PACK
- S_SPEC: pack.ads
- S_BODY: pack.adb
- SEP. FUNCTIONS_TEST: PACK.Functions_TEST.ada
DOT_REPLACEMENT="-"
PACK.Functions_TEST.ada:1:01: warning: invalid file name, contains dot
PACK
- S_SPEC: pack.ads
- S_BODY: pack.adb
- SEP. FUNCTIONS_TEST: PACK-Functions_Test.ada
